src/components/OptionList/components/Checkbox/Checkbox.tsx

```diff
@@ -28,8 +29,28 @@ export function Checkbox({
   role,
 }: CheckboxProps) {
   const id = useUniqueId('Checkbox', idProp);
+  const {newDesignLanguage = false} = useFeatures();
```

Comment:
`newDesignLanguage` is now a required key and defaults to false. You should omit the default assignment here, and in other places where you use this hook.

```suggestion
  const {newDesignLanguage} = useFeatures();
```